controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/aws/AwsEventFetcher.java

```diff
@@ -0,0 +1,25 @@
+package com.yahoo.vespa.hosted.controller.api.integration.aws;
+
+import com.amazonaws.regions.Regions;
+import com.yahoo.config.provision.zone.ZoneId;
+import com.yahoo.vespa.hosted.controller.api.integration.organization.Issue;
+
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public interface AwsEventFetcher {
+    List<CloudEvent> getEvents(Regions awsRegion);
+    Issue createIssue(CloudEvent event);
+
+    default Regions zoneToAwsRegion(ZoneId zoneId) {
```

Comment:
Lets not leak AWS-SDK classes here, change it to `getEvents(ZoneId)` and do the conversion in internal code.